### PR TITLE
added doc for iCub3 joints, Kinematics, robot calibration

### PR DESCRIPTION
**Pull request by davidetome**
📝  As per title this PR adds/updates the following docs  :

* [x] http://wiki.icub.org/wiki/ICub_joints
* [x] http://wiki.icub.org/wiki/ICub_joints_limits
* [x] http://wiki.icub.org/wiki/ICub_Model_naming_conventions
* [x] http://wiki.icub.org/wiki/ICubForwardKinematics
* [x] http://wiki.icub.org/wiki/Vergence,_Version_and_Disparity (related to https://github.com/icub-tech-iit/tickets/issues/386)
* [x] https://redmine.robotology.eu/projects/icub_hw/wiki/ICub3_joints_specifications

It also moves the  `ft-sensors` doc to the upper level.

✅ Once discussed, approved and merged I'll update the old wiki pages with a pointer to the new doc.

cc @pattacini @traversaro @maggia80

### CODE REVIEW

**Review by pattacini — CHANGES_REQUESTED** [review]
Very nicely done @davidetome 👍🏻 

As first feedback, we agreed on getting shot of the PPT files as they've been already ported in MD.

**Comment by davidetome** [comment]
> Very nicely done @davidetome 👍🏻
> 
> As first feedback, we agreed on getting shot of the PPT files as they've been already ported in MD.

❌  `pptx` files removed in the last commit

**Review by pattacini — APPROVED** [review]

**Comment by pattacini** [comment]
Thanks @davidetome!
Let's keep this PR open for a couple of days while I'll be reviewing it.

**Review by pattacini — CHANGES_REQUESTED** [review]
I would move the following topics under HW:
- F/T Sensors
- iCub Kinematics
- iCub 3

Further, I would

**Comment by davidetome** [comment]
> I would move the following topics under HW:
> 
> * F/T Sensors
> * iCub Kinematics
> * iCub 3

done in the last commit 👍🏻

File: mkdocs.yml [hunk 1]
```diff
@@ -85,6 +85,23 @@ nav:
...
+        - iCub Joints Limits: icub_kinematics/icub-joints-limits/icub-joints-limits.md
+        - Robot Configuration: icub_kinematics/icub-robot-configuration/icub-robot-configuration.md 
+        - ICub Model naming conventions: icub_kinematics/icub-model-naming-conventions/icub-model-naming-conventions.md 
+        - Vergence, Version and Disparity: icub_kinematics/icub-vergence-version/icub-vergence-version.md
+        - ICubForwardKinematics: icub_kinematics/icub-forward-kinematics/icub-forward-kinematics.md 
+        - ICubForwardKinematics Arms:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-arms.md 
+        - ICubForwardKinematics Fingers:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-fingers.md 
+        - ICubForwardKinematics Head:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-head.md 
+        - ICubForwardKinematics Legs:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-legs.md 
+        - ICubForwardKinematics Inertial:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-inertial.md
```
Comment (pattacini) [inline mkdocs.yml:102]:
```suggestion
        - iCub FW Kinematics: icub_kinematics/icub-forward-kinematics/icub-forward-kinematics.md 
        - iCub FW Kinematics Arms:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-arms.md 
        - iCub FW Kinematics Fingers:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-fingers.md 
        - iCub FW Kinematics Head:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-head.md 
        - iCub FW Kinematics Legs:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-legs.md 
        - iCub FW Kinematics Inertial:  icub_kinematics/icub-forward-kinematics/icub-forward-kinematics-inertial.md 
```

**Review by pattacini — APPROVED** [review]

**Comment by pattacini** [comment]
Very nicely done @davidetome 🚀 
I would ask @traversaro to give a second look before I'll squash and merge the PR.

**Comment by pattacini** [comment]
Merging, we can always amend the documentation later on.

**Comment by traversaro** [comment]
Thanks, I actually started the review with some notes but it remained in the pending state! I will open separate issues for the improvements, but it is quite nice already!

**Comment by davidetome** [comment]
Updated old pages with a pointer to the new doc

**Pull request merged**